backend/src/api/instances/[id]/music/index.ts

```diff
@@ -1,10 +1,14 @@
 import { Router } from 'opine/mod.ts';
 import { getHandler } from './get.handler.ts';
+import { postHandler } from './post.handler.ts';
+import { patchHandler } from './patch.handler.ts';
 
 const musicRouter = Router({
   mergeParams: true,
 });
 
 musicRouter.get('/', getHandler);
+musicRouter.post('/:action', postHandler);
```

Comment:
@p-fruck Do we try to be RESTful? If yes, then this is "not compliant"

Reply:
@tjarbo yes, we try. How would you suggest implementing it?

Reply:
@p-fruck 
```
GET ../music returns:
{
    status: "playing",
    ...
}
```

```
PATCH ../music with req.body:
{
    status: "stopped",
}
```

Reply:
@tjarbo Ah, I remember :bulb: Approved, that's the way to go!